src/ncp/ncp_base.cpp

```diff
@@ -1651,7 +1651,8 @@ template <> otError NcpBase::HandlePropertySet<SPINEL_PROP_UNSOL_UPDATE_FILTER>(
 
     if (error != OT_ERROR_NONE)
     {
-        WritePropertyValueIsFrame(SPINEL_HEADER_FLAG | SPINEL_HEADER_IID_0, SPINEL_PROP_UNSOL_UPDATE_FILTER);
+        IgnoreReturnValue(
```

Comment:
Any special consideration for the naming of macro of `IgnoreReturnValue`? Shall it be named as `IGNORE_RETURN_VALUE`?

Reply:
This is defined in `code-utils.hpp` and follows the naming model of other macros `VerifyOrExit`, `SuccessOrExit`, `ExitNow` there (code-utils.hpp is OT only header file for internal use).

I know we also have `OT_UNUSED_VARIABLE()` and `OT_UNREACHABLE_CODE()` which are defined in `toolchain.h` (which is an OT public header) and definition can differs for different platforms.